_posts/2024-06-25-create-your-first-python-package-scipy-2024.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: single
+title: "pyOpenSci Workshop: Create Your First Python Package"
+excerpt: "pyOpenSci will be running our `Create Your First Python Package: Make Your Python Code Easier to Share and Use
+` workshop at SciPy 2024. Read on to learn more!"
+author: "Jesse Mostipak"
+permalink: /blog/pyos-workshop-scipy-2024.html
+header:
+  overlay_image: images/blog/2024/june/scipy-2024-workshop.png
+  overlay_filter: rgba(20, 13, 36, 0.8)
+categories:
```

Comment:
let's add an "events" category here. then i can make a page that only lists events as cards similar to the blog post listing... but we can customize the cards to have little calendar icons or something and then have a upcoming events at the top listing with quick overviews.

Reply:
fixed